src/fixture.ts

```diff
@@ -0,0 +1,104 @@
+import { TestRun, Test } from '@saucelabs/sauce-json-reporter';
+
+export type Asset = { name: string, localPath: string };
+export type Assets = Array<Asset>;
+
+/**
+* Provides a facade to help associate a sauce-json-reporter/TestRun with additional TestCafe specific
+* execution metadata (e.g. browser under test, generated assets).
+*/
+class BrowserTestRun {
+    userAgent: string;
+    testRun: TestRun;
+    assets: Assets;
+
+    #browser: string;
+    platform: string;
+
+    constructor(userAgent: string) {
+        this.userAgent = userAgent;
+        this.testRun = new TestRun();
+        // @ts-ignore
```

Comment:
That's a pretty wide ignore that also masks compilation errors. What's actually being ignored here?

Reply:
I moved it to a cast, it more safe.